packages/ddd-toolkit/src/command-bus/local-command-bus.ts

```diff
@@ -10,8 +10,8 @@ export class LocalCommandBus implements ICommandBus {
 
     constructor(
         private logger: ILogger,
-        private readonly retryMaxAttempts = 5,
-        retryInitialDelay = 100,
+        private readonly retryMaxAttempts = 0,
+        retryInitialDelay = 0,
```

Comment:
the delay to 0 forces to be specified in case of retry, maybe here a default might be useful by the way axios-retry might be an interesting example to consider

Reply:
would you leave 100 as _retryInitialDelay_ default?